IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGeotechnicalStratum/DocEntity.xml

```diff
@@ -1,9 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcGeotechnicalStratum" UniqueId="faefb134-3800-4995-b222-b921d7e287bf" BaseDefinition="IfcGeotechnicalElement">
-	<Subtypes>
-		<DocSubtype UniqueId="964a446c-3ce6-44e6-b70b-cc43a1ff39bd" DefinedType="IfcSolidStratum" />
-		<DocSubtype UniqueId="ec4f4f65-4341-4538-8fd3-c067f3bd6986" DefinedType="IfcVoidStratum" />
-		<DocSubtype UniqueId="fb21c163-8d7a-45cf-b517-421f586727b0" DefinedType="IfcWaterStratum" />
-	</Subtypes>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcGeotechnicalStratum" UniqueId="faefb134-3800-4995-b222-b921d7e287bf" BaseDefinition="IfcGeotechnicalElement" EntityFlags="32">
```

Comment:
What is EntityFlags?

Reply:
I guess this was intended to be used for many things but currently it specifies IsAbstract. 0x20 determines non abstract.